pkg/driver/wsl2/vm_windows.go

```diff
@@ -229,3 +229,27 @@ func getWslStatus(instName string) (string, error) {
 
 	return instState, nil
 }
+
+func getSSHAddress(ctx context.Context, instName string) (string, error) {
+	distroName := "lima-" + instName
+	// Ubuntu
+	cmd := exec.CommandContext(ctx, "wsl.exe", "-d", distroName, "bash", "-c", `hostname -I | cut -d ' ' -f1`)
+	out, err := cmd.CombinedOutput()
+	if err == nil {
+		return strings.TrimSpace(string(out)), nil
+	}
+	// Alpine
+	cmd = exec.CommandContext(ctx, "wsl.exe", "-d", distroName, "sh", "-c", `ip route get 1 | awk '{gsub("^.*src ",""); print $1; exit}'`)
+	out, err = cmd.CombinedOutput()
+	if err == nil {
+		return strings.TrimSpace(string(out)), nil
+	}
+	// fallback
+	cmd = exec.CommandContext(ctx, "wsl.exe", "-d", distroName, "hostname", "-i")
+	out, err = cmd.CombinedOutput()
+	if err != nil || strings.HasPrefix(string(out), "127.") {
```

Comment:
https://pkg.go.dev/net#IP.IsLoopback should be used rather than checking the "127." string prefix.

Also add a comment to describe why it is expected to be a loopback address, but it cannot be just hard-coded as 127.0.0.1